src/edit.js

```diff
@@ -2,43 +2,142 @@
 
 	return (
 		<div { ...useBlockProps() }>
-			<InspectorControls>
-				<PanelBody
-					title={ __( 'Settings', 'free-shipping-progress-bar' ) }
-					initialOpen={ true }
-				>
-					<TextControl
-						label={ __(
-							'Free shipping from',
+			<ViewSwitcher
+				label={ __( 'Edit', 'free-shipping-progress-bar' ) }
+				views={ [
+					{
+						value: 'insufficient',
+						name: __(
+							'Insufficient cart totals',
+							'free-shipping-progress-bar'
+						),
+					},
+					{
+						value: 'sufficient',
+						name: __(
+							'Sufficient cart totals',
+							'free-shipping-progress-bar'
+						),
+					},
+				] }
+				defaultView={ 'insufficient' }
+				render={ ( currentView ) => (
+					<BlockErrorBoundary
+						header={ __(
+							'Progress Bar Block Error',
+							'free-shipping-progress-bar'
+						) }
+						text={ __(
+							'There was an error whilst editing the progress bar block. If this problem continues, try re-creating the block.',
 							'free-shipping-progress-bar'
 						) }
-						help={ __(
-							'Provide the value in your store currency.',
+						showErrorMessage={ true }
+						errorMessagePrefix={ __(
+							'Error message:',
 							'free-shipping-progress-bar'
 						) }
-						value={ freeShippingFrom }
-						onChange={ ( value ) =>
-							setAttributes( {
-								freeShippingFrom: value,
-							} )
-						}
-					/>
-				</PanelBody>
-			</InspectorControls>
-			<Block { ...attributes } />
+					>
+						{ currentView === 'insufficient' && (
+							<>
+								<InspectorControls>
+									<PanelBody
+										title={ __(
+											'Settings',
+											'free-shipping-progress-bar'
+										) }
+										initialOpen={ true }
+									>
+										<TextControl
+											label={ __(
+												'Free shipping from',
+												'free-shipping-progress-bar'
+											) }
+											help={ __(
+												'Provide the value in your store currency.',
+												'free-shipping-progress-bar'
+											) }
+											value={ freeShippingFrom }
+											onChange={ ( value ) =>
+												setAttributes( {
+													freeShippingFrom: value,
+												} )
+											}
+										/>
+									</PanelBody>
+								</InspectorControls>
+								<PlainText
+									className="wc-free-shipping-progress-bar__label"
+									value={ labelInsufficientTotals }
+									onChange={ ( value ) =>
+										setAttributes( {
+											labelInsufficientTotals: value,
+										} )
+									}
+								/>
+								{ ! labelInsufficientTotals.includes(
+									'{REMAINING_AMOUNT}'
+								) && (
+									<Notice
+										className="wc-block-checkout__terms_notice"
+										status="warning"
+										isDismissible={ false }
+									>
+										<p>
+											{ __(
+												'Ensure that the label contains the placeholder {REMAINING_AMOUNT}.',
+												'free-shipping-progress-bar'
+											) }
+										</p>
+									</Notice>
+								) }
```

Comment:
I see your point about not update the value because of a language usage, but you don't have to use a static string here. You can just reuse the default value in you have `block.json` which would be already translated. We do exactly that in Terms and Conditions block.